src/constants/addresses.ts

```diff
@@ -36,6 +36,54 @@ const sepoliaAddresses: Addresses = {
   AGGREGATOR_UNISWAP_V3: "",
 };
 
+const arbitrumSepoliaAddresses: Addresses = {
+  LOOKS: "",
+  EXCHANGE_V2: "",
+  TRANSFER_MANAGER_V2: "",
+  WETH: "",
+  ORDER_VALIDATOR_V2: "",
+  REVERSE_RECORDS: "",
+  LOOKS_LP_V3: "",
+  STAKING_POOL_FOR_LOOKS_LP: "",
+  AGGREGATOR_UNISWAP_V3: "",
+};
+
+const arbitrumMainnetAddresses: Addresses = {
+  LOOKS: "",
+  EXCHANGE_V2: "",
+  TRANSFER_MANAGER_V2: "",
+  WETH: "",
+  ORDER_VALIDATOR_V2: "",
+  REVERSE_RECORDS: "",
+  LOOKS_LP_V3: "",
+  STAKING_POOL_FOR_LOOKS_LP: "",
+  AGGREGATOR_UNISWAP_V3: "",
+};
```

Comment:
These Arbitrum are added to resolve the following type error.
<img width="623" alt="type-error" src="https://github.com/LooksRare/sdk-v2/assets/112218419/012b5fbe-8b12-419f-95ee-4c19e0912d8e">